login/style.css

```diff
@@ -1,4 +1,73 @@
-#login-image {
-  width: 100px;
-  height: 100px;
-}
\ No newline at end of file
+
+body {
+  background-color: aliceblue;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+#login-window {
+  position: relative;
+  background-color: antiquewhite;
+  padding: 10px 20px;
+  width: 250px;
+}
+
+.h-centered {
+  margin: 0 auto; /* center horizontally */
+}
+
+#login-window-title {
+  text-align: center;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+#create-new-account {
+  width: 250px;
+  color: blue;
+  text-decoration: underline;
+  text-align: left;
+  cursor: pointer;
+}
+
+#login-window label {
```

Comment:
See how I do not set properties on `label` that would be disastrous, because would change any label anywhere.
This changes only those labels inside and id `#login-window`